src/main/java/gyro/google/iam/ServiceAccountFinder.java

```diff
@@ -0,0 +1,33 @@
+package gyro.google.iam;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.iam.v1.model.ServiceAccount;
+import gyro.core.Type;
+import gyro.google.GoogleFinder;
+
+/**
+ * Query service account.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    service-account: $(external-query google::service-account {})
+ */
+@Type("service-account")
+public class ServiceAccountFinder extends GoogleFinder<Iam, ServiceAccount, ServiceAccountResource> {
+
+    @Override
+    protected List<ServiceAccount> findAllGoogle(Iam client) throws Exception {
+        return client.projects().serviceAccounts().list("projects/" + getProjectId()).execute().getAccounts();
+    }
+
+    @Override
+    protected List<ServiceAccount> findGoogle(Iam client, Map<String, String> filters) throws Exception {
+        throw new UnsupportedOperationException("Finding `service-accounts` with filters is not supported!!");
```

Comment:
This should be supported accepting name and display name
Use the stream api if the api itself does not support this natively